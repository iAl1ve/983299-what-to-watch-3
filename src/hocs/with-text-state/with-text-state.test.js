import React from "react";
import renderer from "react-test-renderer";
import withTextState from "./with-text-state.js";

const MockComponent = () => <div />;
const MockComponentWrapped = withTextState(MockComponent);

it(`withTextState is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped/>
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
