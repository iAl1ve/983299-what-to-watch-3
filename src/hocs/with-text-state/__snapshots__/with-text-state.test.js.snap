// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withTextState is rendered correctly 1`] = `<div />`;
